notebooks/0.feature_selection/nbconverted/0.feature_selection.py

```diff
@@ -157,70 +250,54 @@
 #
 # In this section, we utilize Pycytominer's feature selection function to obtain features that will be used in training our machine learning models.
 
-# In[6]:
+# In[8]:
 
 
 # Applying feature selection using pycytominer
-fs_injury_df = feature_select(
+aligned_cell_injury_fs_df = feature_select(
     profiles=shared_features_df,
     features=shared_feats,
 )
 
 # split meta and feature column names
-fs_injury_meta, fs_injury_feats = utils.split_meta_and_features(fs_injury_df)
-cell_injuries = fs_injury_df["injury_type"].unique()
+fs_injury_meta, fs_injury_feats = utils.split_meta_and_features(
+    aligned_cell_injury_fs_df
+)
+
+# counting number of cell injuries
+cell_injuries = aligned_cell_injury_fs_df["injury_type"].unique()
 
 # display
 print("Number of meta features", len(fs_injury_meta))
 print("Number of features", len(fs_injury_feats))
-print("Shape of fs shared profile", fs_injury_df.shape)
+print("Shape of fs shared profile", aligned_cell_injury_fs_df.shape)
 print("number of cell injury types", len(cell_injuries))
 print(cell_injuries)
-fs_injury_df.head()
-
-
-# Generate encoders and decoders for injuries, and save the file as a JSON file. Additionally, update the feature-selected profile with the injury codes and save it.
-
-# In[7]:
-
-
-# next lets make an injury code
-injury_codes = defaultdict(lambda: {})
-for idx, injury in enumerate(cell_injuries):
-    injury_codes["encoder"][injury] = idx
-    injury_codes["decoder"][idx] = injury
-
-# update shared fs profile with injury codes
-fs_injury_df.insert(
-    0,
-    "injury_code",
-    fs_injury_df["injury_type"].apply(lambda injury: injury_codes["encoder"][injury]),
-)
-
-# now save the injury codes json file
-with open(fs_dir / "injury_codes.json", mode="w") as f:
-    json.dump(injury_codes, f)
+print(aligned_cell_injury_fs_df.shape)
+aligned_cell_injury_fs_df.head()
 
 # save shared feature selected profile
-fs_injury_df.to_csv(
-    fs_dir / "cell_injury_profile_fs.csv.gz",
+aligned_cell_injury_fs_df.to_csv(
```

Comment:
I like the updated naming scheme! (please make sure all other notebooks downstream are updated [note, I haven't checked this], including the visualization notebooks)

Also, if you're introducing two new profile dataframes, please also make sure to delete any legacy `.csv.gz`

Reply:
Will do! That will be my 3rd PR!